Documentation/user-guides/getting-started.md

```diff
@@ -117,8 +117,8 @@ spec:
         - --kubelet-service=kube-system/kubelet
         - -logtostderr=true
         - --config-reloader-image=quay.io/coreos/configmap-reload:v0.0.1
-        - --prometheus-config-reloader=quay.io/coreos/prometheus-config-reloader:v0.23.0
-        image: quay.io/coreos/prometheus-operator:v0.23.0
+        - --prometheus-config-reloader=quay.io/coreos/prometheus-config-reloader:v0.22.2
+        image: quay.io/coreos/prometheus-operator:v0.22.2
```

Comment:
This should not have gotten bumped back down, right?

Reply:
:+1: i stumbled upon this now also